lib/mpl_toolkits/axisartist/floating_axes.py

```diff
@@ -342,6 +343,10 @@ def cla(self):
         self.patch.set_clip_path(orig_patch)
         self.gridlines.set_clip_path(orig_patch)
 
+    def cla(self):
+        """Clear the current axes."""
+        self.clear()
```

Comment:
```suggestion
```

Reply:
Removing the `def cla(self)` because `FloatingAxes` also inherits from Axes with the class factory, as seen on line 359-363 of `lib/mpl_toolkits/axisartist/floating_axes.py`.